README.md

```diff
@@ -1,3 +1,273 @@
+
+The service account credentials will be used to give Gardener access to your base cluster:
+* To create a secret that will be used on the Gardener dashboard to create shoot clusters.
+* The control plane of the seed clusters will use this secret to store the etcd backups of the shoot clusters.
+
+### landscape.etcd
+```yaml
+etcd:
+  backup:
+    type: <gcs|s3>                    # type of blob storage
+    region: (( iaas.region ))         # region of blob storage (default: same as above)
+    credentials: (( iaas.creds ))     # credentials for the blob storage's IaaS provider (default: same as above)
+```
+Configuration of what blob storage to use for the etcd key-value store. If your IaaS provider offers a blob storage you can use the same values for `etc.backup.region` and `etc.backup.credentials` as above for `iaas.region` and `iaas.credentials` correspondingly by using the [(( foo ))](https://github.com/mandelsoft/spiff/blob/master/README.md#-foo-) expression of spiff.
+
+| Field | Type | Description | Example&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Iaas Provider Documentation |
+|:------|:--------|:--------|:--------|:---------|
+|`backup.type`|Fixed value| Type of your blob store. Supported blob stores: `gcs` ([Google Cloud Storage](https://cloud.google.com/storage/)), and `s3` ([Amazon S3](https://aws.amazon.com/s3/)).|`gcs`|n.a.|
+|`backup.region`|IaaS provider specific|Region of blob storage. |`(( iaas.region ))` |[GCP (overview)](https://cloud.google.com/docs/geography-and-regions), [AWS (overview)](https://docs.aws.amazon.com/general/latest/gr/rande.html)|
+|`backup.credentials`|IaaS provider specific|Service account credentials in a provider-specific format. |`(( iaas.creds ))` |[GCP](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys), [AWS](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html#id_users_service_accounts)|
+
+
+### landscape.dns
+```yaml
+dns:
+  type: <google-clouddns|aws-route53>       # dns provider
+  hostedZoneID:                             # hosted zone id for chosen domain
+  credentials:                              # credentials for the dns provider
+```
+Configuration for the Domain Name Service (DNS) provider. If your IaaS provider offers a DNS you can use the same values for `dns.credentials` as for `iaas.creds` above by using the [(( foo ))](https://github.com/mandelsoft/spiff/blob/master/README.md#-foo-) expression of spiff.
+
+| Field | Type | Description | Example |IaaS Provider Documentation
+|:------|:--------|:--------|:--------|:------------|
+|`type`|Fixed value|Your DNS provider. Supported providers: `google-clouddns` ([Google Cloud DNS](https://cloud.google.com/dns/docs/)), and `aws_route53` ([Amazon Route 53](https://aws.amazon.com/route53/)).|`google-clouddns`|n.a.|
```

Comment:
It's `aws-route53`, not `aws_route53`. `azure-dns` is also supported from garden-setup, but not yet from gardener itself (and thus can't be used here). This should change when https://github.com/gardener/gardener/pull/850 is merged.